src/interface.jl

```diff
@@ -2,74 +2,43 @@ using GeometryBasics
-iscatter!(x::AbstractVector, y::AbstractVector; kwargs...) = iviz(pts -> scatter!(pts; kwargs...), Point2fSet(Point2f.(x, y), nothing))
-iscatter!(g, xy::AbstractVector{Point2f}; kwargs...) = iviz(pts -> scatter!(g, pts; kwargs...), Point2fSet(xy, nothing))
-iscatter!(g, x::AbstractVector, y::AbstractVector; kwargs...) = iviz(pts -> scatter!(g, pts; kwargs...), Point2fSet(Point2f.(x, y), nothing))
-
-ilines(f::Function, xmin=0.0, xmax=1.0; kwargs...) = iviz((x, y) -> lines(x, y; kwargs...), Function1D(f, xmin, xmax))
-ilines(y::AbstractVector; kwargs...) = iviz((x, y) -> lines(x, y; kwargs...), Samples1D(eachindex(y), y))
-ilines(x::AbstractVector, y::AbstractVector; kwargs...) = iviz((x, y) -> lines(x, y; kwargs...), Samples1D(x, y))
-ilines(g, f::Function, xmin=0.0, xmax=1.0; kwargs...) = iviz((x, y) -> lines(g, x, y; kwargs...), Function1D(f, xmin, xmax))
-ilines(g, y::AbstractVector; kwargs...) = iviz((x, y) -> lines(g, x, y; kwargs...), Samples1D(eachindex(y), y))
-ilines(g, x::AbstractVector, y::AbstractVector; kwargs...) = iviz((x, y) -> lines(g, x, y; kwargs...), Samples1D(x, y))
-ilines!(f::Function, xmin=0.0, xmax=1.0; kwargs...) = iviz((x, y) -> lines!(x, y; kwargs...), Function1D(f, xmin, xmax))
-ilines!(y::AbstractVector; kwargs...) = iviz((x, y) -> lines!(x, y; kwargs...), Samples1D(eachindex(y), y))
-ilines!(x::AbstractRange, y::AbstractVector; kwargs...) = iviz((x, y) -> lines!(x, y; kwargs...), Samples1D(x, y))
-ilines!(g, f::Function, xmin=0.0, xmax=1.0; kwargs...) = iviz((x, y) -> lines!(g, x, y; kwargs...), Function1D(f, xmin, xmax))
-ilines!(g, y::AbstractVector; kwargs...) = iviz((x, y) -> lines!(g, x, y; kwargs...), Samples1D(eachindex(y), y))
-ilines!(g, x::AbstractRange, y::AbstractVector; kwargs...) = iviz((x, y) -> lines!(g, x, y; kwargs...), Samples1D(x, y))
-
-iheatmap(f::Function, xmin=0.0, xmax=1.0, ymin=0.0, ymax=1.0; kwargs...) = iviz((x, y, z) -> heatmap(x, y, z; kwargs...), Function2D(f, xmin, xmax, ymin, ymax))
-iheatmap(z::AbstractMatrix; kwargs...) = iviz((x, y, z) -> heatmap(x, y, z; kwargs...), Samples2D(1:size(z,1), 1:size(z,2), z))
-iheatmap(x::AbstractRange, y::AbstractRange, z::AbstractMatrix; kwargs...) = iviz((x, y, z) -> heatmap(x, y, z; kwargs...), Samples2D(x, y, z))
-iheatmap(g, f::Function, xmin=0.0, xmax=1.0, ymin=0.0, ymax=1.0; kwargs...) = iviz((x, y, z) -> heatmap(g, x, y, z; kwargs...), Function2D(f, xmin, xmax, ymin, ymax))
-iheatmap(g, z::AbstractMatrix; kwargs...) = iviz((x, y, z) -> heatmap(g, x, y, z; kwargs...), Samples2D(1:size(z,1), 1:size(z,2), z))
-iheatmap(g, x::AbstractRange, y::AbstractRange, z::AbstractMatrix; kwargs...) = iviz((x, y, z) -> heatmap(g, x, y, z; kwargs...), Samples2D(x, y, z))
-
 ### figure-axis-plot container
 
-export repaint
-
 struct FigureAxisPlotEx{T}
   fap::T
   update::Function
-  params::Union{Nothing,Dict{Symbol,Any}}
+  params::Union{Nothing, Dict{Symbol, Any}}
```

Comment:
Note I used JuliaFormatter, so it automatically added explicit return statements and added some additional spacing when declaring types such as here.

Let me know if these should be reverted as well.

Reply:
Prefer code not touched by this PR to be left alone to avoid clutter. For new code you added, I have no objections to return statements of spaces in declarations. Thanks.

Reply:
Done

Reply:
Thanks!